### ISSUE TITLE
Create a subdirectory for Julia translations of the Python functions

### ISSUE THREAD

**Issue by ScottPJones** [AllenDowney/ThinkBayes#10]
It would be nice to have a place for budding Bayesians who are also Julia fanatics to be able to submit as PRs translations of the Python code in the book.

**Comment by AllenDowney** [comment]
Good idea, but we probably want to start with the ThinkBayes2 repo, which is more current (this repo is pretty much frozen in order to stay in sync with the book).

Also I wonder if it would be better to fork a new repo rather than make a directory in this one.  Do you have a preference one way or the other?

**Comment by ScottPJones** [comment]
I have no preference, whatever would work best for you, as you'd probably need to be the final authority on what gets merged (or appoint collaborators on the repo).  I just thought it would be good to have a known place for people to contribute translations for other languages (I of course, want to try to do the code in Julia, simply as part of reading / understanding the book, but maybe somebody else would want to do the same for R, or Lua, for example).

**Comment by AllenDowney** [comment]
Ok, I've created

https://github.com/AllenDowney/ThinkBayes2Translations

Please feel free to send me a PR if you write some code.  Or I can make you
a collaborator if you like.

**Comment by ScottPJones** [comment]
I've got to think of a decent way of representing the structure for the code in thinkbayes.py, maybe splitting that up into smaller chunks, like Pmf, Cdf, etc. so that in Julia, you could say: `using Pmf`, for example.  I'm asking around for help from some of my Julian friends, who are also Python experts, like @ismael-vc ;-)
